### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by eriktaubeneck**
updating the README and Cargo.toml as I'm getting started. 

A few of these benchmarks aren't working for me, and I added notes in the readme. I can delete those and open issues, if that's preferred, but I just wanted to make sure we documented it somewhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,75 @@ the privacy and security goals of the project.
...
+cargo bench --bench oneshot_sort --no-default-features --features="enable-benches descriptive-gate"
+```
+
+**Arithmetic gates**:
+```
+cargo bench --bench oneshot_arithmetic --no-default-features --features="enable-benches descriptive-gate" -- --width 1 --depth 1
+```
+You can adjust the width and depth of the gates at the expense of a longer benchmarking run.
+
+(NOTE: this currently hangs for anything other than `--width 1` for at least 42 minutes.)
```
Comment (akoshelev) [inline README.md:88]: there is a fix for it #754

File: README.md [hunk 2]
```diff
@@ -25,3 +25,75 @@ the privacy and security goals of the project.
...
+
+**Other**:
+```
+cargo bench --bench criterion_arithmetic --no-default-features --features="enable-benches descriptive-gate"
+```
+(NOTE: this currently hangs after "Warming up for 3.0000 s". for at least 6 minutes.)
```
Comment (akoshelev) [inline README.md:94]: Fixed by #754 as well

File: README.md [hunk 3]
```diff
@@ -25,3 +25,75 @@ the privacy and security goals of the project.
...
+
+```
+cargo bench --bench iai_arithmetic --no-default-features --features="enable-benches descriptive-gate"
+```
+(NOTE: this currently fails with error: `Unexpected error while launching valgrind. Error: No such file or directory (os error 2)`.)
```
Comment (akoshelev) [inline README.md:99]: this is expected if you're using MacOS - to the best of my knowledge valgrind does not support it properly. `iai` framework uses it to run the test. We could add a line that this only works on Linux

File: README.md [hunk 4]
```diff
@@ -25,3 +25,75 @@ the privacy and security goals of the project.
 This is very much a work in progress; input is welcome.  However, see our
 [contribution guidelines](./CONTRIBUTING.md) for some important notices
 regarding how to participate in the project.
+
+## Getting Started
+
+### Installation
+
+First, clone this repo. Assuming you have the [GitHub CLI](https://cli.github.com/manual/installation) installed,
```
Comment (akoshelev) [inline README.md:33]: `git clone https://github.com/private-attribution/ipa` works too

**Review by akoshelev — APPROVED** [review]
lets merge #754 and make sure commands work before merging this to main

**Comment by eriktaubeneck** [comment]
Updated this. Should we wait until #754 is merged until we merge this?

**Review by akoshelev — APPROVED** [review]

**Pull request merged**